app/views/needs/show.html.erb

```diff
@@ -10,12 +10,12 @@
 
 <ol class="breadcrumb">
   <li><%= link_to "People in need", contacts_path %></li>
-  <li><%= link_to @need.contact.name , @need.contact %></li>
+  <li><%= link_to @need.contact.name, @need.contact %></li>
   <li>Need</li>
```

Comment:
Heads up that this file will probably have a lot of conflict's with Jaye's overhaul branch. Probably better to merge that branch before this one and resolve the conflict here.

Reply:
agreed - done